exercise-1.js

```diff
@@ -201,83 +352,128 @@
+            name: 'Irina',
+            age: 17,
+            gender: 'female',
+            healthy: 'yes',
+            hobbies: 'yoga',
+            profession: 'manager',
+            education: 'Management Information Systems',
+            learn: function(){
+                console.log(this.name + " is learning JavaScript.");
+            },
+            likes: function(){
+                console.log(this.name + " likes " + this.hobbies + ".");
+            }
+
+        };
+
+        myInfo.learn();
+        myInfo.likes();
 
 
 
         /************************************************************* */
 
-        {
+        
             const year = 2021;
             const nums = [1,2,3,4,5];
             let sum = 0;
             let i = 0;
-            const doubled = [];
+            //const doubled = [];
+            
 
             //Refactor the following statements into expressions
 
 
             // 1.
-            if(year > 2000 && year < 2100) {
-                console.log("welcome to the 21st century");
-            }
+            // if(year > 2000 && year < 2100) {
+            //     console.log("welcome to the 21st century");
+            // }
             
+            year > 2000 && year < 2100 && console.log("welcome to the 21st century");
+            //or
+            year > 2000 && year < 2100 ? console.log("Welcome to the 21st century"): console.log("Not 21st century:(")
+
             // 2.
-            for(let i=0; i<nums.length; i++) {
-                sum += nums[i];
-            }
+            sum = nums.reduce(function(accumulator, val){
+                return accumulator + val;
+            });
             console.log(sum);
+            // for(let i=0; i<nums.length; i++) {
+            //     sum += nums[i];
+            // }
+            // console.log(sum);
+            const array1 = [1, 2, 3, 4];
+
             
+            // 3.   
+            const doubled = nums.map(x => x*2);
+            console.log(doubled); 
             
-            // 3.
-            while(i < nums.length) {
-                doubled.push(nums[i]*2);
-                i++;
-            }
+
+            // while(i < nums.length) {
+            //     doubled.push(nums[i]*2);
+            //     i++;
+            // }
             
-            console.log(doubled);
-        }
+            // console.log(doubled);
+        
 
 
         /************************************************************* */
         // Use array methods to solve the following problems.
 
         const nums = [1,2,3,4,5];
         // Square every number in the array.  Store the squares in a new array.
-
+        
         //your code...
-
+        const squares = nums.map(x => x*x);
+        console.log(squares);
 
 
 
 
         const fivePlus = [1,3,5,7,9,1,3,5,2,3,1,23,4,232,3,4,1,2,2,2,3,4,4,1,12,11,23,3,4,5];
         //Remove all numbers that are less than 5.  Store the results in a new array.
-
+        
         //your code...
```

Comment:
The exercise is asking to remove the numbers below 5 and store the results in a new array.